src/NetAnalyzers/Core/Microsoft.NetCore.Analyzers/MicrosoftNetCoreAnalyzersResources.resx

```diff
@@ -1998,4 +1998,40 @@
   <data name="UseConcreteTypeForParameterMessage" xml:space="preserve">
     <value>Change type of parameter '{0}' from '{1}' to '{2}' for improved performance</value>
   </data>
+  <data name="PreferLengthOverAnyCodeFixTitle" xml:space="preserve">
+    <value>Prefer 'Length' over 'Any()'</value>
+  </data>
+  <data name="PreferLengthOverAnyTitle" xml:space="preserve">
+    <value>Prefer 'Length' over 'Any()'</value>
+  </data>
+  <data name="PreferLengthOverAnyMessage" xml:space="preserve">
+    <value>Prefer a 'Length' check over 'Any()' to avoid the need for devirtualization</value>
```

Comment:
NIT:  For what meaning the `devirtualization` is used? It seems used for handling virtual functions
```suggestion
    <value>Prefer a 'Length' check over 'Any()' to avoid unnecessary enumeration</value>
```